frontend/src/main/components/Commons/CommonsForm.js

```diff
@@ -85,6 +85,7 @@ function CommonsForm({ initialCommons, submitAction, buttonLabel = "Create" }) {
                         required: "Cow price is required",
                         min: {
                             value: 0.01,
+
```

Comment:
Extra space, on mobile so I don't know if it helps with the readability on PC though

Reply:
Removed